managed/services/minio/service.go

```diff
@@ -111,7 +124,80 @@ func (s *Service) RemoveRecursive(ctx context.Context, endpoint, accessKey, secr
 	return nil
 }
 
-func newClient(endpoint, accessKey, secretKey string) (*minio.Client, error) {
+// List provides an abstraction over the minio API to list all objects in the bucket
+// It scans path with prefix and returns all files with given suffix.
+// Both prefix and suffix can be omitted.
+func (m *Client) List(ctx context.Context, prefix, suffix string) ([]FileInfo, error) {
```

Comment:
I think it should be method of `Service`, not `Client`. And it seems that we don't need `Client` at all.

Reply:
We'll replace `Service` with `Client` since it more accurately describes the functionality.